src/playerInfo.h

```diff
@@ -45,6 +45,7 @@ class PlayerInfo : public MultiplayerObject
     uint32_t main_screen_control = 0;
     int32_t ship_id;
     string name;
+    string last_password;
```

Comment:
there is a possible server password, gm password and ship control code, it's now not clear which this is used for.

Reply:
renamed to last_ship_password